xml/signals/B&O-1980/index.shtml

```diff
@@ -50,21 +52,25 @@ To use the CPL files you need to set up the signal as three heads.  The top 3 ma
 <p>
 I am not an expert on the B&amp;O semaphore signals.  These definitions come straight from the rule books and may have been used in conjunction with CPLs.  They lack a lot of the necessary aspects and indications to be a complete signal system on their own.
 
-  
+
 </body>
 
 
 <p>
-See the 
+See the
 <a href="aspects.xml">aspect page</a>.
 
 <p>
 Signal Mast definitions:
 <ul>
 <li><a href="appearance-CPL-High.xml">Color Position Light Mast Signals</a>
+<li><a href="appearance-CPL-High-shortblock.xml">Color Position Light Mast Signals Short Block</a>
 <li><a href="appearance-CPL-Dwarf.xml">Color Position Light Dwarf Signals</a>
+<li><a href="appearance-CPL-Dwarf-shortblock.xml">Color Position Light Dwarf Signals Short Block</a>
 <li><a href="appearance-CPL-High-permissive.xml">Permissive Color Position Light Mast Signals</a>
+<li><a href="appearance-CPL-High-permissive-shortblock.xml">Permissive Color Position Light Mast Signals Short Block</
```

Comment:
This line has `</` instead of `</a>` at the end.  That makes the list format oddly, and breaks the next link.  Not really a big functional deal, happy to merge anyway, but it would be nice to fix.  (Also, `Tripple` two lines down has an extra p)